src/views/Editor.vue

```diff
@@ -20,16 +20,16 @@
 					</b-form-select>
 				</b-input-group>
 
-				<b-input-group size="sm" class="w-25 mx-1" prepend="owner">
+				<b-input-group size="sm" prepend="Owner">
```

Comment:
is the case change from `owner` to `Owner` intended change

Reply:
Yes. A bit out of scope but makes the UI a bit more consistent, and I was changing the line anyway.

Reply:
wanted to confirm that this was not an accidental change